Create ui-c8y-1023-12-0-automatically-pause-resume-on-tab-visibility-change-10327-graft-release-cd.md (#4000)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-12-0-automatically-pause-resume-on-tab-visibility-change-10327-graft-release-cd.md

---
date: ""

> title: Auto-refresh automatically paused and resumed on tab visibility change

Reviewer: ```suggestion
title: Auto-refresh is automatically paused and resumed on tab visibility change
```

> title: Automatically pause and resume global context auto-refresh on tab visibility change

Reviewer: ```suggestion
title: Auto-refresh automatically paused and resumed on tab visibility change
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65227
version: 1023.12.0
---

> Global context now pauses auto-refresh updates when the browser tab is hidden and resumes when visible. This optimizes resource usage while ensuring the data is refreshed upon return.

Reviewer: ```suggestion
The global context now pauses auto-refresh updates when the browser tab is hidden, and resumes updates when the browser tab is visible. This optimizes resource usage while ensuring the data is refreshed upon return.
```
